src/main/java/model/data/DataModel.java

```diff
@@ -92,9 +96,13 @@ public DataTable get(int index) {
 	 * @return The DataTable with the given name or null if none was found.
 	 */
 	public DataTable getByName(String name) {
-		return dataTables.stream()
+		return tables.stream()
 				.filter(
 						x -> x.getName().equals(name)
 				).findFirst().get();
 	}
+
+	public ObservableList<DataTable> getObservableList() {
```

Comment:
Ja, toen ik om 0:30 iets werkend moest krijgen, dan gaat werkend boven fatsoenlijk. Nu dien jij de pull request in (en het was altijd al jouw branch), dus heb jij er de verantwoordelijkheid over.